Tribler/Core/Session.py

```diff
@@ -123,6 +123,19 @@ def set_and_create_dir(dirname, setter, default_dir):
                 permidmod.save_keypair(self.keypair, pairfilename)
                 permidmod.save_pub_key(self.keypair, pubfilename)
 
+            multichain_pairfilename = scfg.get_multichain_permid_keypair_filename()
+
+            if os.access(multichain_pairfilename, os.F_OK):
```

Comment:
I suppose that you're checking here whether the `multichain_pairfilename` file exists on the disk? I think it's a bit nicer to use `os.path.exists(multichain_pairfilename)` for that.